docs/internals/deprecation.txt

```diff
@@ -15,6 +15,9 @@ about each item can often be found in the release notes of two versions prior.
 See the :ref:`Django 1.9 release notes<deprecated-features-1.9>` for more
 details on these changes.
 
+* Support for querying related object by model name in query lookup
```

Comment:
put this on line 143 (this section is for changes deprecated in 1.9 as it says above)